<launch>
  
  <node pkg="apriltags2_ros" type="apriltags2_ros_continuous_node" name="apriltags2_ros_continuous_node" clear_params="true" >
    <!-- Remap topics from those used in code to those on the ROS network -->
    <remap from="image_rect" to="/camera/rgb/image_rect_color" />
    <remap from="camera_info" to="/camera/rgb/camera_info" />

    <param name="camera_frame" type="str" value="camera_rgb_optical_frame" />
    <param name="publish_tag_detections_image" type="bool" value="true" />      <!-- default: false -->

    <rosparam command="load" file="$(find tams_bartender_recognition)/config/apriltags2/settings.yaml"/>
    <rosparam command="load" file="$(find tams_bartender_recognition)/config/apriltags2/tags.yaml"/>
  </node>
</launch>
